Show build status in UI

mayankkapoor opened the pull request:
Now user can see the build status and build finished time in the UI.

sorenisanerd commented on data/public_repos/brandon/brandon/buildlogs/aaSemble_python-aasemble.django.git/aaSemble_python-aasemble.django.git_0.1a0+1.log:
```diff
@@ -0,0 +1,68 @@
+ -- aaSemble Package Builder <pkgbuild@aasemble.com>  Thu, 03 Dec 2015 18:57:49 +0000
+
+
+Starting source Package Build
```
I think adding this was unintentional?

mayankkapoor committed:
Show build status in UI

- Update builds.html to show build status
- Change order of links on left menu

sorenisanerd commented on aasemble/django/apps/buildsvc/templates/buildsvc/html/builds.html:
```diff
@@ -22,6 +24,14 @@
+              {% else %}
+                  Running
+              {% endif %}
+          </a></td>
```
Builds from before we started tracking build_finished will show as "Running". Are you ok with that, or do you have thoughts on how to avoid it?

mayankkapoor committed:
Replace build status with duration

Now showing build duration, status can be inferred. Duration is more
useful info.

mayankkapoor commented:
@sorenh ready for review. I've replaced build status with build duration. Duration is more useful info to show in UI. Status can be inferred. Builds from before will show "None".

mayankkapoor committed:
Merge branch 'master' into show-build-status-in-UI

mayankkapoor committed:
Merge branch 'master' into show-build-status-in-UI
